dev-learning/Startup.cs

```diff
@@ -17,11 +19,20 @@ public Startup(IConfiguration configuration)
         public IConfiguration Configuration { get; }
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services)
+       /* public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
             _ = services.AddDbContext<Models.UserContext>(opt =>
                 opt.UseInMemoryDatabase("Users"));
+        }*/
```

Comment:
Why this is left? I guess it should be removed and not commented out if it's not needed anymore.